src/wp-includes/css/admin-bar.css

```diff
@@ -138,6 +138,11 @@ html:lang(he-il) .rtl #wpadminbar * {
 	padding: 0 8px 0 7px;
 }
 
+#wpadminbar .quicklinks > ul > li,
```

Comment:
I could see this being an optional include as this was simply proposed at https://wordpress.org/support/topic/awesome-plugin-works-great-13/ while it then didn't seem to be harmful & seemed to help at least one person out there.